platform/setup.gradle

```diff
@@ -2,13 +2,15 @@ import org.apache.tools.ant.taskdefs.condition.Os
 
 def jdk17_win_64 = 'https://api.adoptopenjdk.net/v3/binary/version/jdk-17.0.5+8/windows/x64/jdk/hotspot/normal/adoptopenjdk'
 def jdk17_linux_64 = 'https://api.adoptopenjdk.net/v3/binary/version/jdk-17.0.5+8/linux/x64/jdk/hotspot/normal/adoptopenjdk'
-def jdk17_mac_64 = 'https://api.adoptopenjdk.net/v3/binary/version/jdk-17.0.5+8/mac/x64/jdk/hotspot/normal/adoptopenjdk'
+def jdk17_mac_64 = 'https://download.bell-sw.com/java/17.0.6+10/bellsoft-jdk17.0.6+10-macos-aarch64-full.tar.gz'

```

Comment:
We prefer to use the same distribution, that is adoptopenjdk. It is more widespread and more tested.

Reply:
AdoptOpenJDK always produces the error:
[Thread-0/ERROR] [STDERR] Mar 31, 2023 2:04:17 PM com.sun.javafx.application.PlatformImpl startup
WARNING: Unsupported JavaFX configuration

Reply:
Does this block any of the application functions? Because this happens on each MCreator launch IIUC.

Reply:
There are issues on ARM machines with adoptium but I as far as I know this error message only has to do with load locations of JFx.
Additionally, adoptium includes a bunch of nonessential content and folders and I would have to recompile the mach-o

Reply:
This error is normal. Adoptium should be used, we have moved from liberica as adoptium is more widespread, tested, and has better security coverage

Reply:
Got it.